app/src/main/res/values/strings.xml

```diff
@@ -58,6 +58,7 @@
     <string name="text_language_parallel_description">Share GodTools with someone in their native language.</string>
     <string name="text_language_warning">Not every tool is available in every language.</string>
     <string name="label_language_none">None</string>
+    <string name="label_language_search">Enter Parallel Language</string>
```

Comment:
This will probably be adjusted again, but for now let's change this to "Filter Languages"